### PR TITLE
Add alternative firmware v0.51 for Enbrighten/GE ZW4002/55258

### PR DESCRIPTION
**Pull request by jascoproducts**
Adding latest beta firmware for 55258.

### CODE REVIEW

**Comment by jascoproducts** [comment]
New version link will need integrity hash added.

**Comment by AlCalzone** [comment]
The tests don't show the expected hash. I guess that's because of the comment.

Can you generate it as described here?
https://github.com/zwave-js/firmware-integrity#usage

**Comment by AlCalzone** [comment]
Not sure when I will be able to use a computer again, the little ones take almost all my time. You can try removing the comment `//`s and using a correctly formatted but incorrect integrity hash. Then the PR checks should show the correct one.

**Comment by AlCalzone** [comment]
Looks good. Do you want to model the upgrade paths like on the draft PR or just keep it as is and have users on the 0.x version ignore the "update"

**Comment by jascoproducts** [comment]
Just did. Hoping it works as intended.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
You'll want to limit the existing 5.x entry to `firmwareVersion >= 5.0` (stable) and add another copy with the condition `firmwareVersion < 1.0` and beta channel to allow switching back.

The way it is now doesn't prevent the update to 5.x from being shown to 0.x users.

**Comment by AlCalzone** [comment]
Sorry for the back and forth btw. If I had the time I'd do the changes myself.

**Comment by jascoproducts** [comment]
Done! No worries. Still trying to figure this out.

File: firmwares/enbrighten-ge/ZW4002_55258.json [hunk 1]
```diff
@@ -10,10 +10,33 @@
...
 		{
+			"$if": "firmwareVersion >= 5.0",
 			"version": "5.50",
 			"changelog": "1. Original Release Firmware",
 			"url": "https://raw.githubusercontent.com/jascoproducts/firmware/9b5520f5056f7d3fce3fa6adc28bdaf27f8d0a68/zwave/Enbrighten-GE/ZW4002/55258%20-%20In-Wall%20Smart%20Fan%20Speed%20Control%2C%20500S/5.50/ZW4002_Enbrighten-GE_55258_5.50.otz",
 			"integrity": "sha256:59ae4c4f749d3ad22b1d490e9dd0afe855bbd9914337f37a41beb56d62c638b2"
+		},
+		{
+			"$if": "firmwareVersion === 0.51",
+			"version": "5.50",
```
Comment (AlCalzone) [inline firmwares/enbrighten-ge/ZW4002_55258.json:21]: I'd add
```suggestion
			"version": "5.50",
			"channel": "beta",
```
so this doesn't permanently get shown as an update to the users of the alternate firmware.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**